qcodes/dataset/data_set.py

```diff
@@ -497,6 +515,29 @@ def add_parameter(self, spec: ParamSpec):
                                   'Please use DataSet.set_interdependencies '
                                   'instead.')
 
+    def set_parent_dataset_links(self, links: List[Link]) -> None:
```

Comment:
Should this be a property setter?

Reply:
Yes, that's probably more natural. The same holds true for `set_interdependencies`, but we can change that later.